src/libs/OptionsListUtils.js

```diff
@@ -696,7 +694,16 @@ function getCategoryListSections(categories, recentlyUsedCategories, selectedOpt
     }
 
     const selectedOptionNames = _.map(selectedOptions, (selectedOption) => selectedOption.name);
-    const filteredRecentlyUsedCategories = _.filter(recentlyUsedCategories, (category) => !_.includes(selectedOptionNames, category.name));
+    const filteredRecentlyUsedCategories = _.map(
+        _.filter(recentlyUsedCategories, (category) => !_.includes(selectedOptionNames, category)),
+        (category) => {
+            const categoryObject = _.find(categories, (c) => c.name === category);
```

Comment:
`_.find` is for lists but the `categories` stored in Onyx are an object keyed by category name, so I think grabbing the category object can be easier than this.